rsocket/transports/tcp/TcpDuplexConnection.h

```diff
@@ -24,6 +24,9 @@ class TcpDuplexConnection : public DuplexConnection {
 
   void setInput(yarpl::Reference<DuplexConnection::Subscriber>) override;
 
+  // Only to be used for observation purposes.
+  folly::AsyncSocket* getTransport();
```

Comment:
Nit: I'd name it as `getSocket()` instead.

Reply:
Let me keep it as getTransport for now. It will make all the functions to have the same name through RSocket, Proxygen, Thrift etc.